pgcontents/tests/test_pgcontents_api.py

```diff
@@ -374,6 +411,9 @@ def teardown_class(cls):
         super(PostgresContentsFileCheckpointsAPITest, cls).teardown_class()
         cls.td.cleanup()
 
+    def test_checkpoints_move_with_file(self):
+        pass
```

Comment:
So this test fails for this case because the FileCheckpoints class is not recognizing any checkpoints when renaming a file. See https://github.com/jupyter/notebook/blob/master/notebook/services/contents/filecheckpoints.py#L98-L99. I'm not sure what to do about this without making upstream changes, but I can imagine that method wanting to list out all checkpoints for the given path if the path is a directory.

Reply:
Hmm, yeah, that looks like it's a bug upstream. I opened https://github.com/quantopian/pgcontents/issues/68 to track this. Can you add a comment pointing to that issue? I'll push an issue and/or bugfix upstream.

Reply:
Ya I'll add a comment.